setup.py

```diff
@@ -244,7 +246,7 @@ def configuration(parent_package='', top_path=None):
         extras_require=extras_require,
         python_requires='>=3.8',
         packages=setuptools.find_packages(exclude=['doc', 'benchmarks']),
-        include_package_data=True,
+        include_package_data=False,
```

Comment:
Can you confirm that test files are still included with this flag?

Reply:
I think it works because we happen to have __init__.py files in our test directories. Otherwise, I am not sure if we would need to list all of these in a `package_data` entry.